examples/schrodinger_semigroup1d.cpp

```diff
@@ -0,0 +1,144 @@
+#include "MRCPP/MWFunctions"
+#include <MRCPP/MWOperators>
+#include <MRCPP/Printer>
+#include <MRCPP/Timer>
+#include "operators/TimeEvolutionOperator.h"
+
+
+
+std::complex<double> free_particle_analytical_solution(double x, double x0, double t, double sigma);
+
+const auto min_scale = 0;
+const auto max_depth = 25;
+
+const auto order = 7;
+const auto prec = 1.0e-7;
+
+int main(int argc, char **argv)
+{
+    auto timer = mrcpp::Timer();
+
+    // Initialize printing
+    auto printlevel = 0;
+    mrcpp::Printer::init(printlevel);
+    mrcpp::print::environment(0);
+    mrcpp::print::header(0, "Building operator");
+
+    // Initialize world in the unit cube [0,1]
+    auto basis = mrcpp::LegendreBasis(order);
+    auto world = mrcpp::BoundingBox<1>(min_scale);
+    auto MRA = mrcpp::MultiResolutionAnalysis<1>(world, basis, max_depth);
+
+    mrcpp::print::footer(0, timer, 2);
+
+    // Time moment:
+    double t = 0.001;
+    int finest_scale = 8;
+
+    mrcpp::TimeEvolutionOperator<1> ReExp(MRA, prec, t, finest_scale, false);
```

Comment:
We name classes with the first letter capitalized, but objects are not capitalized, and possibly with longer names
Example:
`class MyClass`
`MyClass myObject`
So in your case it should be `TimeEvolutionOperator realExpansion`

I see many other places where this convention is for the time being not followed. It is not critical but it is important that the coding style is uniform throughout the mrcpp code. This not only has aesthetic reasons, but it makes the code more readable and quicker to understand if the conventions are followed.

Reply:
Can you rewrite those as being `realExponent` instead of `ReExp`? Then they are both more readable and in line with the style used in mrchem/mrcpp. There might be exceptions, yes but the coding style of both mrchem and mrcpp is as I described. But we can take this up tomorrow at the meeting, in connection with code documentation.

Reply:
How do you like this name:
class MPI_FuncVector
?